tailnet/telemetry_internal_test.go

```diff
@@ -0,0 +1,128 @@
+package tailnet
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"tailscale.com/tailcfg"
+
+	"github.com/coder/coder/v2/tailnet/proto"
+)
+
+func TestTelemetryStore(t *testing.T) {
```

Comment:
Should test that hashes are unique per `newTelemetryStore()` with the same IPs/hostnames